src/wasm/wasm.cpp

```diff
@@ -245,6 +245,17 @@ const char* getExpressionName(Expression* curr) {
       return "array.set";
     case Expression::Id::ArrayLenId:
       return "array.len";
+    case Expression::Id::RefAsId:
```

Comment:
It looks like `RefIs` and `RefAs` are the only expression classes that change their names based on their contents. Is there a particular reason that they are special here? Also, I'm not sure what this function is actually useful for, given all our printing facilities. Can we remove it in a separate change?

Reply:
Yeah, we should remove `getExpressionName`, good point. I'll do that in my very next PR (if it it can't be removed, I'll figure out how to fix things so it makes sense).